cmd/root.go

```diff
@@ -47,6 +47,8 @@ func newRootCmd() *cobra.Command {
 
 	rootCmd.PersistentFlags().StringVarP(&cfgFile, "config", "c", "", "config file (default is $HOME/.flyte/config.yaml)")
 
+	configAccessor.InitializePflags(rootCmd.PersistentFlags())
```

Comment:
Can we exclude certain flags from being registered .
For eg : These might cause confusion with persistent flags of -p, -d and -o defined in flytectl

```
      --root.domain string                        Specified the domain to work on.
      --root.output string                         Specified the output type. (default "TABLE")
      --root.project string                        Specifies the project to work on.
```

Reply:
we can remove the pflags generation for that... I've updated the pr